controllers/packetmachine_controller.go

```diff
@@ -362,6 +353,11 @@ func (r *PacketMachineReconciler) reconcile(ctx context.Context, machineScope *s
 	machineScope.SetProviderID(dev.ID)
 	machineScope.SetInstanceStatus(infrav1.PacketResourceStatus(dev.State))
 
+	if err := r.PacketClient.EnsureNodeBGPEnabled(dev.ID); err != nil {
+		// Do not treat an error enabling bgp on machine as fatal
+		return ctrl.Result{RequeueAfter: time.Second * 20}, fmt.Errorf("failed to enable bpg on machine %s: %w", machineScope.Name(), err)
+	}
+
```

Comment:
@cprivitere Should I make this configurable with an env var? For example:

```go
if os.env("EIP_MANAGEMENT" == "KUBE_VIP" {
	if err := r.PacketClient.EnsureNodeBGPEnabled(dev.ID); err != nil {
		// Do not treat an error enabling bgp on machine as fatal
		return ctrl.Result{RequeueAfter: time.Second * 20}, fmt.Errorf("failed to enable bpg on machine %s: %w", machineScope.Name(), err)
	}
}
```

Reply:
This seems a good place to start, if nothing else. Let's start with it and after I do some deployments I can comment if there's more tweaking to do.